### DISCUSSION TITLE
overwrite one digital value / harcoded value for one particular digit

### DISCUSSION THREAD

**Discussion by weghorst** [jomjol/AI-on-the-edge-device#2820] [Q&A]
Is it possible to overwrite / pre-define one particular digit value? I have a reflection that i cannot avoid on the first digit which will probably change every 3-4 years. a pre-defined or overwritten value would help to solve the issue

<img width="1012" alt="Bildschirmfoto 2024-01-22 um 12 15 06" src="https://github.com/jomjol/AI-on-the-edge-device/assets/67224764/f1c5e2dd-c6d3-4ba9-b56a-7208fab0cbb9">

**Comment by caco3** [comment]
Why not just keep that digit away?
in general one is only interested in relative changes of the meter anyway.
Alternatively, that would be a task for post processing, eg. in Homeassistant.

**Comment by caco3** [comment]
And usually you can get rid of the reflection by rotating the whole ESP32-CAM.
